Template/Variable/MatomoConfigurationVariable.php

```diff
@@ -72,6 +75,7 @@ public function getParameters()
                 $field->validators[] = new NotEmpty();
                 $field->validators[] = new CharacterLength(0, 500);
                 $field->validate = function ($value) use ($matomoUrl, $url) {
+                    $value = trim($value);
```

Comment:
Need to add this here if someone pastes something like this " 3", it will throw an error and the user might not understand what is wrong easily as we check for numeric and throw an exception if not